Adressing inconsistencies, mistakes and references (#34)
hyprwm/hyprland-wiki: pages/Nix/_index.md

[flake](https://github.com/hyprwm/Hyprland/blob/main/flake.nix).
{{< /hint >}}

# Install and configure Hyprland on NixOS

Make sure to check out the options of the
[Nix module](https://github.com/hyprwm/Hyprland/blob/main/nix/module.nix).

> Do note that the Nixpkgs Hyprland package is not actively maintained, and may be outdated

Reviewer: missed a period at the end

Make sure to check out the options of the
[Nix module](https://github.com/hyprwm/Hyprland/blob/main/nix/module.nix).

> Do note that the Nixpkg for Hyprland is no actively maintained, and may be outdated.

Reviewer 2: ```suggestion
Do note that the Nixpkgs Hyprland package is not actively maintained, and may be outdated.
```
